### PR TITLE
Adjust to renamed BusNotifications

### PR DESCRIPTION
**Pull request by timbussmann**
Adjust docs, snippets and samples to renaming of `BusNotifications` to `Notifications`.

### CODE REVIEW

**Comment by timbussmann** [comment]
@Particular/docs-maintainers please review the changes. Will update the package as soon as it's merged & pushed to myget.

File: samples/logging/notifications/Version_6/Sample/Sample.csproj [hunk 1]
```diff
@@ -37,8 +37,9 @@
     <UseVSHostingProcess>false</UseVSHostingProcess>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="NServiceBus.Core">
-      <HintPath>..\..\..\..\..\packages\NServiceBus.6.0.0-unstable1705\lib\net452\NServiceBus.Core.dll</HintPath>
+    <Reference Include="NServiceBus.Core, Version=6.0.0.0, Culture=neutral, PublicKeyToken=9fc386479f8a226c, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\..\..\..\NServiceBus\binaries\NServiceBus.Core.dll</HintPath>
```
Comment (SimonCropp) [inline samples/logging/notifications/Version_6/Sample/Sample.csproj]: This seems wrong. Assuming u will fix before removing wip

File: nservicebus/errors/subscribing-to-error-notifications.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Error notifications are available for several events.
 
 This API was added in Version 5.1.
 
-These event are exposed via the `BusNotifications` class that can be injected via DI.
+These event are exposed via the `BusNotifications` (NServiceBus Version 5) or `Notifications` (NServiceBus Version 6) class that can be injected via DI.
```
Comment (indualagarsamy) [inline nservicebus/errors/subscribing-to-error-notifications.md]: See version language here: https://github.com/particular/docs.particular.net#version-language

> These event are exposed via the `BusNotifications` (NServiceBus Version 5) or `Notifications` (NServiceBus Version 6) class that can be injected via DI.

How about:
These events are exposed via the `BusNotifications` class for Versions 5 and below.  For Versions 6 and above, these events are exposed via the `Notifications` class. This class can be injected via DI.

File: samples/logging/notifications/sample.md [hunk 3]
```diff
@@ -35,9 +35,9 @@ snippet: customSLR
 
 ## Plugging to the API
 
-The notifications API is exposed via the BusNotifications class that can be injected via DI.
+The notifications API is exposed via the `BusNotifications` (NServiceBus Version 5) or `Notifications` (NServiceBus Version 6) class that can be injected via DI.
```
Comment (indualagarsamy) [inline samples/logging/notifications/sample.md]: See version language here: https://github.com/particular/docs.particular.net#version-language

> These event are exposed via the `BusNotifications` (NServiceBus Version 5) or `Notifications` (NServiceBus Version 6) class that can be injected via DI.

How about:
The notifications API is exposed via the `BusNotifications` class for Versions 5 and below.  For Versions 6 and above, the API is available via the `Notifications` class. This class can be injected via DI.

File: nservicebus/upgrades/5to6.md [hunk 4]
```diff
@@ -632,7 +633,7 @@ snippet: 5to6-MsmqSubscriptionAuthorizer
 
 ## Synchronous request-response (callbacks)
 
-The synchronous request-response feature, also known as callbacks, has been moved from the NServiceBus core to the separate Nuget package [NServiceBus.Callbacks](https://www.nuget.org/packages/NServiceBus.Callbacks/). That package must be used in order to use the callback functionality in Version 6. 
+The synchronous request-response feature, also known as callbacks, has been moved from the NServiceBus core to the separate Nuget package [NServiceBus.Callbacks](https://www.nuget.org/packages/NServiceBus.Callbacks/). That package must be used in order to use the callback functionality in Version 6.
```
Comment (indualagarsamy) [inline nservicebus/upgrades/5to6.md]: >The synchronous request-response feature, also known as callbacks, has been moved from the NServiceBus core to the separate Nuget package [NServiceBus.Callbacks](https://www.nuget.org/packages/NServiceBus.Callbacks/). That package must be used in order to use the callback functionality in Version 6.

The synchronous request-response feature, also known as callbacks, has been moved from the NServiceBus core to a separate Nuget package called [NServiceBus.Callbacks](https://www.nuget.org/packages/NServiceBus.Callbacks/). To use the callback functionality in Versions 6 and above, please use this new package.

Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: @timbussmann u need to fix your md editing tool to not auto fix trailing spaces. Your prs keep on coming through as though you have touched lines that you have not really touched.

File: samples/logging/notifications/sample.md (hunk 3 above)
Comment (mikeminutillo) [inline samples/logging/notifications/sample.md]: Do we use the DI shorthand elsewhere in doco? It's probably better to be explicit to say "using the container" and link it to http://docs.particular.net/nservicebus/containers/

Comment (indualagarsamy) [inline samples/logging/notifications/sample.md]: @mikeminutillo - good point. Here's the mod:
The notifications API is exposed via the BusNotifications class for Versions 5 and below. For Versions 6 and above, the API is available via the Notifications class. This class can be injected [using the container](/nservicebus/containers/.)

File: samples/logging/notifications/Version_6/Sample/Sample.csproj (hunk 1 above)
Comment (timbussmann) [inline samples/logging/notifications/Version_6/Sample/Sample.csproj]: yes will fix as soon the unstable package is available :+1:

File: nservicebus/upgrades/5to6.md (hunk 4 above)
Comment (timbussmann) [inline nservicebus/upgrades/5to6.md]: is there a reason for the trailing spaces or why to keep them?

Is this configurable in Atom?

Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: i dont mind them being clean up. i do it when i am doing a full review of the pages. just saying that if u do a PR and u touch a line it looks like you have edited it. similar to if in a cs file u change one line then submit the PR with a full code cleanup.

So perhaps do a cleanup + direct commit before doing a PR.

Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: or perhaps just open and save all the files in atom to do a gobal cleanup?

Comment (timbussmann) [inline nservicebus/upgrades/5to6.md]: > or perhaps just open and save all the files in atom to do a gobal cleanup?

that's what I'm just thinking about. Going to check whether there is an easy way to do a global cleanup.

File: nservicebus/errors/subscribing-to-error-notifications.md (hunk 2 above)
Comment (timbussmann) [inline nservicebus/errors/subscribing-to-error-notifications.md]: no need to mention lower versions than v5 since this was added in 5.1?

**Comment by timbussmann** [comment]
@Particular/docs-maintainers updated & rebased.

I didn't change this sentence: These event are exposed via the `BusNotifications` (NServiceBus Version 5) or `Notifications` (NServiceBus Version 6) class
since both mentioning versions below v5 or above v6 doesn't make sense since this api was introduced in v5 and will be removed after v6. I changed DI to a link to containers as suggested though.

**Comment by SimonCropp** [comment]
merged manually

**Pull request closed without merging**